components/src/core/components/Chart/PieChart.vue

```diff
@@ -145,6 +145,7 @@ export default defineComponent({
       aspectRatio:
         typeof props.aspectRatio === 'number' ? props.aspectRatio : 1,
       cutout: props.cutout,
+      borderWidth: props.data.filter(item => item.value).length > 1,
```

Comment:
bug fix?

Reply:
not formally. this was something came up in review. 

**Old behaviour**
![image](https://user-images.githubusercontent.com/12880059/188078967-6730ce55-0b1a-4c86-b35b-ead5ab5f8645.png)

**New behaviour**
![image](https://user-images.githubusercontent.com/12880059/188079108-4c554868-17c9-4617-8dcf-90c7aef19f8e.png)